tool-box/schema-detection/runtime_linking.py

```diff
@@ -0,0 +1,91 @@
+def use_df():
+    import sys
+    import os
+    import digdag
+    sys.path.append('/home/td-user/.local/lib/python3.6/site-packages')
+    os.system(f"{sys.executable} -m pip install --user pandas")
+    os.system(f"{sys.executable} -m pip install --user -U git+https://github.com/treasure-data/pytd#egg=pytd[spark]")
+    os.system(f"{sys.executable} -m pip install --user time")
+    os.system(f"{sys.executable} -m pip install --user pyyaml")
+    os.system(f"{sys.executable} -m pip install --user json")
+    os.system(f"{sys.executable} -m pip install --user requests")
+    os.system(f"{sys.executable} -m pip install --user digdag")
```

Comment:
Remove this pip command. Instead of it, please add
`import digdag`

Reply:
With `digdag/digdag-python:3.7`, you don't need to use `--user` option anymore.

The following packages are installed:

- pandas
- requests
- pytd (old version, so would be better to use `pip install`

also should remove the following packages:
- json (Python standard module)
- time (Python standard module)
- digdag (not on PyPI)

You should write like the following:

```
os.system(f"{sys.executable} -m pip install -U pytd==0.8.0")
os.system(f"{sys.executable} -m pip install pyyaml")
```